spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -43,4 +71,16 @@ public void setModules(String[] modules) {
 		return modules;
 	}
 
+	@PostConstruct
+	public void deduceModuleArguments() throws Exception {
```

Comment:
What about the equivalent of yaml above + an explicit `modulesKeys` which is a list of labels:

```yaml
modules
  log:
    coordinates: org.springframework.cloud.stream.module:log-sink:1.0.0.BUILD-SNAPSHOT
    arguments:
      thing: 1
  time:    // module "label"
    coordinates: org.springframework.cloud.stream.module:time-source:1.0.0.BUILD-SNAPSHOT
    arguments:
      quizz: fix
```

```MODULES_KEYS=time,log```